iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/TieLineImpl.java

```diff
@@ -136,6 +136,7 @@ public void remove() {
 
         network.getListeners().notifyAfterRemoval(id);
         removed = true;
+        //TODO p0 and q0 must be recomputed on the 2 dangling lines.
```

Comment:
I think we can remove this todo. In some use cases we don't want P0 & Q0 to be recomputed. When you're removing the tie line to link another dangling line for instance. And we might want to keep the original P0 & Q0 instead of recomputing it.